staging/src/k8s.io/apiserver/pkg/apis/audit/types.go

```diff
@@ -33,14 +34,17 @@ const (
 	// LevelRequest provides Metadata level of auditing, and additionally
 	// logs the request object (does not apply for non-resource requests).
 	LevelRequest Level = "Request"
-	// LevelResponse provides Request level of auditing, and additionally
+	// LevelRequestResponse provides Request level of auditing, and additionally
 	// logs the response object (does not apply for non-resource requests).
-	LevelResponse Level = "Response"
+	LevelRequestResponse Level = "RequestResponse"
```

Comment:
The idea was that you the user would always want metadata, and it's just the request/response part that doesn't have a well defined "ordering". Anyhow, let's follow up elsewhere.